TaskHero/CreateStepsViewController.swift

```diff
@@ -87,6 +91,11 @@ class CreateStepsViewController: UIViewController {
         tableView.deleteRows(at: [indexPath], with: .fade)
         tableView.endUpdates()
     }
+    
+    private func resetNavigationStackToNewTask() {
+        let root = self.storyboard?.instantiateViewController(withIdentifier: "CreateTaskViewController")
```

Comment:
Okay cool - let's keep as is so we can still make tasks for our testing, but def change it later.